crates/xline/src/server/auth_server.rs

```diff
@@ -55,11 +65,13 @@ impl AuthServer {
     where
         T: Into<RequestWrapper>,
     {
-        let token = get_token(request.metadata());
-        let wrapper = RequestWithToken::new_with_token(request.into_inner().into(), token);
-        let cmd = command_from_request_wrapper(wrapper);
-
-        let res = self.client.propose(&cmd, use_fast_path).await??;
+        let auth_info = match get_token(request.metadata()) {
+            Some(token) => Some(self.auth_store.verify(&token)?),
```

Comment:
If we access the storage layer here, can we ensure that permission verification is always valid?

Reply:
This is just authentication, the command will be authorized when it enters the state machine, and if the auth info is invalid, it will be intercepted during authorization

Reply:
If it should be invalid (e.g. expired when checked on the leader), but is recognized as valid on a machine with incorrect system time, should the JWT validation be separated from checking permissions?

Reply:
Now it's separated, it's just JWT validation here, the permission check will be done in the `prepare` method of the state machine